README.md

```diff
@@ -173,7 +173,7 @@ Check that the given element has no content.
 ```javascript
 const { getByTestId } = render(<View testID="empty" />);
 
-expect(getByTestId('empty')).toBeEmpty();
+expect(getByTestId('empty')).toBeEmptyElement();
 ```
 
```

Comment:
```suggestion
---
**NOTE**
`toBeEmptyElement()` matcher has been renamed from `toBeEmpty()` because of the naming conflict with Jest Extended export with the [same name](https://github.com/jest-community/jest-extended#tobeempty).

---
```